charts/csi-unity/values.yaml

```diff
@@ -6,6 +6,20 @@
 # Examples : "v2.8.0" , "nightly"
 version: "v2.8.0"
 
+images:
+  # "driver" defines the container image, used for the driver container.
+  driver: dellemc/csi-unity:v2.8.0
+  # CSI sidecars
+  attacher: registry.k8s.io/sig-storage/csi-attacher:v4.4.0
+  provisioner: registry.k8s.io/sig-storage/csi-provisioner:v3.6.0
+  snapshotter: registry.k8s.io/sig-storage/csi-snapshotter:v6.3.0
+  resizer: registry.k8s.io/sig-storage/csi-resizer:v1.9.0
+  registrar: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.9.0
+  healthmonitor: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.10.0
+
+  # CSM sidecars
+  podmon: dellemc/podmon:v1.7.0
```

Comment:
Should podmon.image [here ](https://github.com/dell/helm-charts/blob/a35aebf2e32f5a83c0e560a45256dbad1fcf3d75/charts/csi-unity/values.yaml#L211) be removed in favor of this change?

Reply:
Good catch ! Yes, we need to remove that entry

Reply:
Fixed.